model_compression_toolkit/qat/keras/quantizer/quantization_builder.py

```diff
@@ -40,20 +40,22 @@ def get_activation_quantizer_holder(n: common.BaseNode,
     Returns:
         A ActivationQuantizationHolder layer for the node activation quantization.
     """
-    if _is_qat_applicable(n, DEFAULT_KERAS_INFO):
-        _, activation_quantizers = quantization_builder(n,
-                                                        qat_config,
-                                                        DEFAULT_KERAS_INFO)
-
-        # Holder by definition uses a single quantizer for the activation quantization
-        # thus we make sure this is the only possible case (unless it's a node with no activation
-        # quantization, which in this case has an empty list).
-        if len(activation_quantizers) > 0:
-            assert len(activation_quantizers) == 1, f'ActivationQuantizationHolder supports a ' \
-                                                    f'single quantizer but {len(activation_quantizers)} quantizers ' \
-                                                    f'were found for node {n}'
-            return ActivationQuantizationHolder(activation_quantizers[0])
-    return None
+    # if _is_qat_applicable(n, DEFAULT_KERAS_INFO):
+    _, activation_quantizers = quantization_builder(n,
```

Comment:
Also, is this another remainder from the previous "adding holder to QAT" PR?

Reply:
Yes. Removing the flag in the model builder breaks the QAT so it has to be removed here as well.